app/proto/jail.proto

```diff
@@ -26,20 +26,25 @@ service Jail {
   }
 
   rpc AcceptTOS(AcceptTOSReq) returns (JailInfoRes) {
-    // Accept to TOS
+    // Accept TOS
   }
 
   rpc SetLocation(SetLocationReq) returns (JailInfoRes) {
     // Set the user's location
     // If you're not jailed, do this with api.UpdateProfile!
   }
+
+  rpc AcceptCommunityGuidelines(AcceptCommunityGuidelinesReq) returns (JailInfoRes) {
```

Comment:
Please document how I, as a third-party app developer, is going to use this method, for example by giving a link to the text to be accepted and how new versions are announced.